### ISSUE TITLE
Negative loss

### ISSUE THREAD

**Issue by EricaCoppolillo** [claws-lab/MisinfoCorrect#1]
Hi.
I have just a question about the loss function: in the paper, the loss is defined as L = -r * log(p(\hat(c)|m)), where r is the reward and p is the probability to generate \hat(c) given the message m. 
Now, to the best of my comprehension, the second term should be negative (since p is a probability) and r should be positive (or zero). Hence, the final loss should be positive as well. 
In the code, instead, another "-" is added to the second term, resulting in a final negative loss.
Is it correct? If yes, what is the rationale?
Thank you in advance for your kind response.

**Comment by binghe2727** [comment]
Hi,
Thanks for your interest in our wok. For your question,

1. "Is it correct?": the reward may be negative sometimes if a certain reward is negative in Equation 1.
2. "what is the rationale?": 1) the rationale to add "log" is to avoid the underflow/overflow problem when computing the loss; 2) the reasoning for adding the "negative -" is to use the gradient descent rather than the gradient ascent. 

Best,
Bing
